live-examples/css-examples/pseudo-class/modal.css

```
@@ -10,3 +10,7 @@ button {
   border: 2px solid burlywood;
   border-radius: 5px;
 }
+
+dialog p{
+  color: initial;
+}
```

```suggestion
dialog p {
  color:  #000;
}
```
initial sets it to canvasText, which is white in dark mode and black in light mode. Setting it to black when the background is off-white works.

---

You're right! I must have overlooked something when making the changes. Your changes to the code produce make the desired effect.